api/src/main/java/edu/cornell/mannlib/vitro/webapp/application/VitroHomeDirectory.java

```diff
@@ -52,6 +73,219 @@ public String getDiscoveryMessage() {
 		return discoveryMessage;
 	}
 
+	/**
+	 * Populates VIVO home directory with files required to run.
+	 * 
+	 * NOTE: Will not overwrite any modified files on redeploy.
+	 */
+	public void populate() {
+		File vhdDir = getPath().toFile();
+
+		if (!vhdDir.isDirectory() || vhdDir.list() == null) {
+			throw new RuntimeException("Application home dir is not a directory! " + vhdDir);
+		}
+
+		Map<String, String> digest = untar(vhdDir);
+
+		writeDigest(digest);
+	}
+
+	/**
+	 * A non-destructive untar process that returns checksum digest of tarred files.
+	 * 
+	 * Checksum digest can be manually created with the following command.
+	 * 
+	 * `find /vivo/home -type f | grep -E "^/vivo/home/bin/|^/vivo/home/config/|^/vivo/home/rdf/" | xargs md5sum > /vivo/home/digest.md5`
```

Comment:
Yes, that might be useful. Depending of the status of Vitro home creation process, I suppose FATAL, ERROR, WARN and INFO levels should be used

Reply:
@chenejac can we standardize the format of the digest file? [Parallel File Systems – Enabling insights from our data](https://parallelstorage.com/) 4/7/2023 11:04AM CT? If the application generates a POSIX standard file, we should be good to go. I suppose we should write a test for the digest file against the POSIX standard... Can we create an issue for this instead of any further changes to this PR?

Reply:
Ahhh. home sweAT HOME https://dl.acm.org/doi/abs/10.5555/121348

Reply:
> @chenejac can we standardize the format of the digest file? [Parallel File Systems – Enabling insights from our data](https://parallelstorage.com/) 4/7/2023 11:04AM CT? If the application generates a POSIX standard file, we should be good to go. I suppose we should write a test for the digest file against the POSIX standard... Can we create an issue for this instead of any further changes to this PR?

Sure, I think it is good idea, and I think it can be resolved in the new PR. @wwelling can you please open a ticket/github issue? Thanks